features/plain_features/panics.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Panic handling
```

Comment:
Does this need adding to CI? It looks like we run each feature separately so it won't get picked up automatically

Reply:
It runs along with the rest of the "plain" features in the "maze tests - plain Go apps" step. It took many attempts to get it working. 😓 😅

Reply:
Ah you're right — I missed that it was in a directory!